src/features/permissions/logics/permissionLogic.ts

```diff
@@ -0,0 +1,47 @@
+const PERMISSIONS_KEY = "permissions";
+
+const getPermissions = (): string | string[] | null => {
```

Comment:
also - mixing `null` and `undefined` is not preferred as you use optional `permissions?: ` in other places. If `null` it should return it `undefined`

Reply:
the `logics` folder is meant to have functions that are performing some sort of action with the data, where `helpers` are more thought of functions that are doing some repetitive task, which can be generalized, and maybe even benefit other parts of the project. I think of helpers as "lodash" functions.

For example, for the permissions the functions in the logics folder are getting, setting and resetting the session storage.

I agree that having a folder with a single file can seem a bit unnecessary, but the idea is that it will be easier to add new files if the folders are already in place, and jumping between features will look consistent. 

In my new commit, you can see how me and @geekonthemove have refactored it a bit so it maybe makes more sense. 

But let's definitely discuss it further 😊